frontend/src/app/[locale]/search/QueryProvider.tsx

```diff
@@ -3,7 +3,7 @@
 import { useSearchParams } from "next/navigation";
 import { createContext, useCallback, useMemo, useState } from "react";
 
-interface QueryContextParams {
+export interface QueryContextParams {
```

Comment:
Is there a reason to export this? Didn't see it called elsewhere

Reply:
good catch, this is left over from a previous idea. Removing.